win32/deluge-bbfreeze.py

```diff
@@ -3,12 +3,11 @@
 import shutil
 import sys
 
-import gtk
-
-import bbfreeze.recipes
+import bbfreeze
 import deluge.common
 import icon
-from bbfreeze import Freezer
+from bbfreeze import Freezer, recipes  # NOQA
+from gi.repository import Gtk  # NOQA
```

Comment:
because other wise isort/flake fails. honeslty though im not sure why they are there but i just updated the import for the old import gtk so i just updated the import and left it alone.